docs/apis-tools/zeebe-api/technical-error-handling.md

```diff
@@ -0,0 +1,36 @@
+---
+id: technical-error-handling
+title: "Technical error handling"
+sidebar_position: 3
+slug: /apis-tools/zeebe-api/technical-error-handling
+description: "Learn more about business logic errors as a result of request processing logic."
+---
+
+In the documentation above, the documented errors are business logic errors.
```

Comment:
@christinaausley post-merge feedback is the worst, I'm sorry! But I noticed while investigating https://github.com/camunda/camunda-docs/pull/3095#issuecomment-1878971608 & the broken build that this extracted page starts off with `In the documentation above`....which doesn't make sense anymore.

Reply:
It is referring to the sections above this in the sidebar, but agreed that this can be reworded to "in the sections above this document" or something of the like.